src/components/DatePicker/DatePicker.vue

```diff
@@ -58,14 +58,15 @@
         </div>
```

Comment:
I'm not sure why this is showing up as a change. There were some changed made to the DatePicker in this branch that weren't needed so I checked out what was in main so this should match what's in main